pkg/providers/vsphere/config/template-cp.yaml

```diff
@@ -208,39 +208,46 @@ spec:
         spec:
           containers:
           - args:
-            - start
+            - manager
             env:
             - name: vip_arp
+              value: "true"  
```

Comment:
capv 1.1.0 updated to use the newwer kube-vip, here is there snippet of args:

```            
- name: cp_enable
  value: "true"
- name: vip_interface
  value: ${VIP_NETWORK_INTERFACE=""}
- name: address
  value: ${CONTROL_PLANE_ENDPOINT_IP}
- name: port
  value: "6443"
- name: vip_arp
  value: "true"
- name: vip_leaderelection
  value: "true"
- name: vip_leaseduration
  value: "15"
- name: vip_renewdeadline
  value: "10"
- name: vip_retryperiod
  value: "2"
```

Did you remove vip interface?

Reply:
yeah, kube-vip added a new feature where it auto-detects the right interface so we don't need to hardcode it to `eth0` 
It was added in the 0.4.0 release https://github.com/kube-vip/kube-vip/releases/tag/v0.4.0